js/adapt-contrib-glossaryView.js

```diff
@@ -3,62 +3,65 @@ define([
   './adapt-contrib-glossaryItemView'
 ], function(Adapt, GlossaryItemView) {
 
-  var GlossaryView = Backbone.View.extend({
+  class GlossaryView extends Backbone.View {
 
-    className: 'glossary',
-
-    events: {
-      'keyup .js-glossary-textbox-change': 'onInputTextBoxValueChange',
-      'input .js-glossary-textbox-change': 'onInputTextBoxValueChange',
-      'paste .js-glossary-textbox-change': 'onInputTextBoxValueChange',
-      'change .js-glossary-checkbox-change': 'onInputTextBoxValueChange',
-      'click .js-glossary-cancel-btn-click': 'onCancelButtonClick'
-    },
+    className() {
+      return 'glossary';
+    }
 
-    itemViews: null,
-    prevScrollPos: 0,
-    isSearchActive: false,
+    events() {
+      return {
+        'keyup .js-glossary-textbox-change': 'onInputTextBoxValueChange',
+        'input .js-glossary-textbox-change': 'onInputTextBoxValueChange',
+        'paste .js-glossary-textbox-change': 'onInputTextBoxValueChange',
+        'change .js-glossary-checkbox-change': 'onInputTextBoxValueChange',
+        'click .js-glossary-cancel-btn-click': 'onCancelButtonClick'
+      };
+    }
 
-    initialize: function() {
+    initialize() {
+      this.itemViews = null;
+      this.prevScrollPos = 0;
+      this.isSearchActive = false;
       this.listenTo(Adapt, 'remove drawer:closed', this.remove);
       this.setupModel();
       this.onScroll = _.debounce(this.onScroll.bind(this), 200);
+      this.onInputTextBoxValueChange = _.debounce(this.onInputTextBoxValueChange.bind(this), 200);
       this.render();
-    },
+    }
 
-    checkForTermToShow: function() {
+    checkForTermToShow() {
       if (this.$el.data('termtoshow') === undefined) return;
 
-      for (var i = 0, count = this.itemViews.length; i < count; i++) {
-        var itemView = this.itemViews[i];
-        if (itemView.model.get('term').toLowerCase() === this.$el.data('termtoshow').toLowerCase()) {
-          Adapt.trigger('glossary:descriptionOpen', itemView.model.cid);
-          break;
-        }
+      for (let i = 0, count = this.itemViews.length; i < count; i++) {
+        const itemView = this.itemViews[i];
+        if (!itemView.model.get('term').toLowerCase() === this.$el.data('termtoshow').toLowerCase()) return;
```

Comment:
`if (itemView.model.get('term').toLowerCase() !== this.$el.data('termtoshow').toLowerCase()) return;`